tests/shared/models/available-drives.spec.js

```diff
@@ -354,9 +386,11 @@ describe('Model: availableDrives', function () {
               device: '/dev/sdc',
               name: 'USB Drive',
               size: 9999999,
-              mountpoint: '/mnt/bar',
-              system: false,
-              protected: false
+              mountpoints: [ {
```

Comment:
I have the impression the linter will complain about this.

Reply:
Quite the opposite; it wants a space there... :/